### PR TITLE
Review all keybindings

### PR DESCRIPTION
**Pull request by aadcg**
# Description

Some cosmetic fixes for those who suffer from OCD.

Some opinionated changes.

Please follow each commit! 

# Discussion

Commits c0855ef9 and 56377b34.

# Checklist:
Everything in this checklist is required for each PR.  Please do not approve a PR that does not have all of these items.  

- [x] I have pulled from master before submitting this PR
- [x] My code follows the style guidelines for Common Lisp code
  - See [Norvig & Pitman's Tutorial on Good Lisp Programming Style (PDF)](https://www.cs.umd.edu/~nau/cmsc421/norvig-lisp-style.pdf) and [Google Common Lisp Style Guide](https://google.github.io/styleguide/lispguide.xml).
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer (the peer review to approve a PR counts.  The reviewer must download and test the code)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by aadcg** [comment]
To align the mapping: select the keyscheme-map s-exp, M-x align-regexp, "'", done.

**Comment by Ambrevar** [comment]
There are some good changes here, however I disagree with the alignment change.  

While aligning is OK (even encouraged?) in documentation because it's user-facing, the source code is a bit more lower-level.

For us, maintaining this alignment is going to be a burden.  For instance I use Lispy and and I tend to let it automatically format the s-exps I work on.  This won't work well with alignment.

If we really want to show aligned source code to the user, I'd be more on the side of adding a special-case for the `define-keyscheme-map` s-exp in `function-lambda-string`, or specialize the pretty-printer if that would work here.

**Comment by aadcg** [comment]
> I disagree with the alignment change

I get the point, and I'll revert that.

**Comment by aadcg** [comment]
Take another look @aartaka @Ambrevar.

File: source/mode/prompt-buffer.lisp [hunk 1]
```diff
@@ -21,119 +21,117 @@ listed and run with `return-selection-over-action' (bound to \"M-return\" by
...
-       "G" 'select-last
+       "C-j" 'select-next
        "g g" 'select-first
-       "J" 'select-next-source
+       "G" 'select-last
+       "C-b" 'select-previous-page
+       "C-f" 'select-next-page
        "K" 'select-previous-source
+       "J" 'select-next-source
        ;; Same as with C-j.
```
Comment (aartaka) [inline source/mode/prompt-buffer.lisp:107]: Move comment too?

Comment (aartaka) [reply source/mode/prompt-buffer.lisp:107]: I mean, the line below it was moved. Quite likely it belonged there.

**Review by aartaka — APPROVED** [review]

File: source/mode/prompt-buffer.lisp (hunk 1 above)
Comment (aadcg) [reply source/mode/prompt-buffer.lisp:107]: Nice catch! Fixed by 6b2d3fd3.

**Comment by aadcg** [comment]
I'm merging. Thanks everyone.

**Pull request merged**